parts/ch11.tex

```diff
@@ -12,15 +12,17 @@ \subsection{Новый взгляд на мысль}
 
 \subsection{Интенсиональность и экстенсиональность}
 
-Мысль должна зависеть \emph{от отражения действительности аппаратурой мозга} . В предыдущих главах мы разработали формальные системы, отражающие области математической действительности с помощью символов. До какой степени подобные формальные системы могут служить моделями обращения мозга с идеями?
+Мысль должна зависеть \emph{от отражения действительности аппаратурой мозга}. В предыдущих главах мы разработали формальные системы, отражающие области математической действительности с помощью символов. До какой степени подобные формальные системы могут служить моделями обращения мозга с идеями?
 
 В системе \textbf{pr} и затем в других, более сложных системах мы видели, как значение, в ограниченном смысле этого слова, возникает из изоморфизма, соотносящего типографские символы с числами, арифметическими действиями и отношениями, а строчки типографских символов --- с высказываниями. В мозгу нет никаких типографских символов, но есть кое-что получше: активные элементы, которые могут хранить информацию, а также передавать её и получать новую информацию от других активных элементов. Таким образом, у нас есть активные символы вместо пассивных типографских символов. В мозгу правила смешаны с самими символами, в то время как на бумаге символы --- это статичные единицы, а правила находятся у нас в голове. Благодаря строгости формальных систем, которые мы до сих пор рассматривали, читатель может заключить, что изоморфизм между символами и реальными вещами --- это жесткое взаимно однозначное соответствие, что-то вроде ниток, соединяющих марионетку с ведущей её рукой. Однако важно понимать, что это вовсе не так. В той же ТТЧ понятие «пятьдесят» может быть выражено различными символами, скажем:
+\begin{center}
+    ((SSSSSSS{\large0} $\cdot$ SSSSSSS{\large0})+(S{\large0} $\cdot$ S{\large0}))
```

Comment:
wtf is \large0 ?
Make it a math-block instead of text-center:
```
\[
    ((\mathrm{SSSSS0} \cdot \mathrm{SSSSS0})........)
\]
```

Reply:
\large0 was my attempt to make this formula look beautiful, but I'll try this method.